### ISSUE TITLE
Destination input

### ISSUE THREAD

**Issue by gradzevicius** [Esri/Trek2There#50]
There is a need to set destination not only in Decimal degrees (Latitude/Longitude) but also in Meters.

**Comment by defiantgoat** [comment]
@gradzevicius Yes, this is definitely in the backlog to add to the application.

**Comment by marikavertzonis** [comment]
see MyUSNG coordinate parser for this: `Coordinate.parse`

**Comment by marikavertzonis** [comment]
@gradzevicius - I know its been sometime since this request, but we are considering how this could be implemented. Your original request stops at asking for '...Meters'. Are you expecting that you would be selecting a projected coordinate system to work with, from a list? Would support for just WGS 84 Web Mercator (as an example) be enough?

**Comment by dominikargast** [comment]
@marikavertzonis test with 3.0.7

**Comment by marikavertzonis** [comment]
testing with 3.0.8 on windows phone 10, windows desktop 10, iphone 6 (ios 10.3.3), trimble tdc (android 6)
 - when you manually enter coordinates in one format, then switch to another, they appear as expected (good)
 - when you copy / paste coordinates (see #56), the contents of the fields are not updated until you switch format (bad)
 - when you delete coordinates in one format, then switch to another format, the previous coordinates are returned (bad)

**Comment by marikavertzonis** [comment]
testing with 3.0.9 on ios and windows, retesting the elements of the previous comment
 - when you manually enter coordinates in one format, then switch to another, they appear as expected (good)
 - when you copy / paste coordinates, the contents of the fields are updated (good!)
 - when you delete coordinates in one format, then switch to another format, the previous coordinates are returned (same as in previous testing). based on the behavior in the last point (ie: 'paste' effectively pastes over the content of all forms of the coordinates), then i would expect that delete to also delete all forms of the coordinates.
